org.kde.PlatformInputContexts.MaliitSailfishOS.appdata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.kde.PlatformInputContexts.MaliitSailfishOS</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Maliit plugin for SailfishOS</name>
+  <summary>Keyboard plugin used in Sailfish OS for Flatpak applications</summary>
+  <description>
+    <p>Keyboard plugin for Sailfish OS that supports it's native input.</p>
```

Comment:
nitpick, "it's" should be "its" here, without the apostrophe.

Reply:
Thanks, fixed now.